content/cpp/concepts/math-functions/terms/fmax/fmax.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'fmax()'
+Description: 'Returns the larger of two arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`fmax()`** function returns the larger of two arguments.
+
+## Syntax
+
+```pseudo
+fmax(x, y)
+```
+
+The parameters `x` and `y` are usually a numeric [data type](https://www.codecademy.com/resources/docs/cpp/data-types) ranging from `-INFINITY` (negative) to `INFINITY` (positive). If one of the parameters is `NAN`, then the other parameter is returned.
+
+## Example
+
+The following example uses the `fmax()` function to find the larger value between `3` and `5`:
+
+```cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+  double x = 3;
+  double y = 5;
+  double result;
+  result = fmax(x, y);
+  std::cout << "The larger value between " << x << " and " << y << " is " << result << "\n";
```

Comment:
Perhaps we can add a period here before the new line (and edit the output to include it)